sim/battle-stream.ts

```diff
@@ -85,20 +77,19 @@ class BattleStream extends Streams.ObjectReadWriteStream {
 		if (this.battle) this.battle.sendUpdates();
 		let deltaTime = Date.now() - startTime;
 		if (deltaTime > 1000) {
+			// tslint:disable-next-line:no-console
```

Comment:
Why isn't `no-console` just globally disabled? PS uses it kind of freely.

Reply:
Because I had coped the config from the client.